testv2/testv2_management/test_management_client.py

```
@@ -706,110 +722,582 @@ def test_token_manager_error_handling(self, mock_token_manager_class, mock_api_c
+                warnings.simplefilter("always")
+                result = client.create_feature_flag(create_request)
+                
+                assert len(w) == 1
+                assert issubclass(w[0].category, DeprecationWarning)
+                assert "create_feature_flag()" in str(w[0].message)
+                
+            assert result is not None
+
+    @patch.object(ManagementTokenManager, 'get_access_token', return_value="fake_token")
+    @patch.object(ManagementTokenManager, '__init__', return_value=None)
+    def test_deprecated_update_feature_flag_wrapper(self, mock_tm_init, mock_get_token):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_tm_init`

(ARG002)